README.md

```diff
@@ -46,6 +84,12 @@ If you wish to contribute to this plugin, please use the usual fork-branch-push-
 
 [Plugin Development](https://github.com/inspec/train/blob/master/docs/dev/plugins.md) is documented on the `train` project on GitHub.
 
+### Unit tests
+
+Run `bundle exec rake test:unit` to run the unit tests.
+
 ### Testing changes against a Windows Machine
 
-TODO
\ No newline at end of file
+Install Vagrant and VirtualBox. Check the Vagrantfile to verify it references a Windows 2016 evaluation VagrantBox to which you have access.
+
+Then run `bundle exec rake test:integration` . There are sub-tasks you can use to run only the integration tests; see `rake -aT`.
```

Comment:
Would you run `rake -aT` to see more sub-task information or is there a specific link you had in mind here?

Reply:
The command lists all tasks.

Reply:
How's this then? 
```suggestion
Then,  run `bundle exec rake test:integration`. There are sub-tasks you can use to run only the integration tests; to see a list of all tasks, run `rake -aT`.
```